packages/fxa-admin-panel/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "1.235.1",
   "description": "FxA Admin Panel",
   "scripts": {
-    "build-postcss": "postcss src/styles/tailwind.css -o src/styles/tailwind.out.css",
-    "build:client": "tsc --build ../fxa-react && tsc --build && NODE_ENV=production npm run build-postcss && SKIP_PREFLIGHT_CHECK=true PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false rescripts build",
+    "build-css": "tailwindcss -i ./src/styles/tailwind.css -o ./src/styles/tailwind.out.css",
```

Comment:
See [this comment](https://github.com/mozilla/fxa/pull/13461#discussion_r908575832), not sure why it's not showing up under files changed.